packages/react-composites/tests/browser/common/constants.ts

```diff
@@ -29,7 +29,7 @@ export const IDS = {
   holdPage: 'hold-page',
   moreButton: 'call-with-chat-composite-more-button',
   callPage: 'call-page',
-  unsupportedBrowserIcon: 'unsupportedBrowserIcon',
-  unsupportedBrowserLink: 'unsupportedBrowserLink',
+  unsupportedEnvironmentIcon: 'unsupportedEnvironmentIcon',
```

Comment:
Minor: I can't help wanting to make this consistent with the other id names with 'unsupported-environment-icon' and  'unsupported-environment-link'